apps/mobile/src/components/FilterOffersScreen/components/LocationSearch/index.tsx

```diff
@@ -108,31 +70,23 @@ function LocationSearch({
         <Input
           autoFocus
           value={inputValue}
-          onChangeText={setInputValue}
+          onChangeText={(value) => {
+            onInputValueChange(value)
```

Comment:
Better to not create anonymous function when it's not needed: `onChangeText={onInputValueChange}`